nodes/nomos-node-api/src/http/da.rs

```diff
@@ -14,6 +18,12 @@ pub(crate) type DaMempoolService = MempoolService<
     CertDiscriminant,
 >;
 
+pub type DataAvailability = DataAvailabilityService<
```

Comment:
better to reuse the type in nomos-node

Reply:
@al8n, refactor for this will be dona later on? Let's open an issue for tracking it please!

Reply:
Yeah, will open a new PR soon for this.